tests/jobsTest.js

```diff
@@ -0,0 +1,69 @@
+const webdriver = require('selenium-webdriver');
+const { until } = require('selenium-webdriver');
+const { By } = require('selenium-webdriver');
+const assert = require('assert');
+const chai = require('chai');
+const expect = chai.expect;
+
+let driver;
+
+describe('Jobs', function () {
+    this.timeout(0);
+
+    before(async function () {
+        this.driver = await new webdriver.Builder().forBrowser('chrome').build();
+    });
+
+    after(async function () {
+        this.driver.quit();
+    })
+
+    it('should render job spec page with job roles details when role id is valid', async function () {
+        await this.driver.get('http://localhost:3001/jobs/job-roles-spec/2');
+        
+        const title = await this.driver.getTitle();
+        expect(title).equal('Specification for Job Role');  
+
+        var roleIDText = await this.driver.findElement(webdriver.By.id('roleID')).getText();
```

Comment:
Not sure why var is being used in this file instead of const, is there a reason behind it?

Reply:
I have addressed this now